app/install.php

```diff
@@ -35,7 +35,7 @@ function initTranslate(): void {
 	}
 
 	if (!in_array(Minz_Session::paramString('language'), $available_languages, true)) {
-		Minz_Session::_param('language', 'en');
+		Minz_Session::_param('language', Minz_Translate::DEFAULT_LANGUAGE);
```

Comment:
Would it be better to have it in a separate PR while this PR is about the API information page?